inspirehep/modules/disambiguation/core/ml/models.py

```diff
@@ -76,3 +101,174 @@ def fit(self):
 
     def predict(self, X):
         return self.estimator.predict(X)
+
+
+class DistanceEstimator(object):
+    def __init__(self, ethnicity_model_path):
+        self.ethnicity_estimator = EthnicityEstimator()
+        self.ethnicity_estimator.load_model(ethnicity_model_path)
```

Comment:
Come to think of it, why are we doing this? The constructor of `DistanceEstimator` should get an instance of an `EthnicityEstimator`, not know how to build one! It's the responsibility of the caller to know how to instantiate `EthnicityEstimator`.